SpatialGDK/Extras/schema/tombstone.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component Tombstone {
+    id = 9982;
+
+    bool is_dead = 1;
```

Comment:
Do we need this bool? What about just adding the tombstone component once a startup actor *is* dead?

Reply:
Yep longer term that's what I was thinking too.  In the first iteration, i'd like to avoid Dynamic Components, as they introduce additional risk - can chat about specifics if you like.

Reply:
What risk do we think they currently add?

Reply:
It's probably a misreading of the Dynamic Component docs on my part related to how ops behave with the feature enabled.  NWX is trying to keep risk to an absolute minimum for their end of milestone (next friday).

I'll go ahead with the Dynamic Component solution now for Master, and keep the non-dynamic version in NWX until we're past the milestone.

Reply:
The implementation is now converted to use Dynamic Components.